src/components/Counter.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="row justify-center q-mt-md q-mb-md">
+    <q-btn-group rounded>
+      <q-btn icon="remove" text-color="brown-14" :disable="getCounter===0" @click="counter(false)" />
+      <q-btn :label="getCounter.toString()" />
+      <q-btn icon="add" text-color="brown-14" @click="counter(true)" />
+    </q-btn-group>
+  </div>
+</template>
+<script>
+import {mapGetters, mapMutations} from 'vuex'
+export default {
+  computed: {
+    ...mapGetters('menu', ['getCounter']),
+  },
+  methods: {
+    ...mapMutations({
+      setCounter(dispatch, payload) {
+        return dispatch('menu/setCounter', payload)
+      },
+    }),
+    counter(type) {
+      var counter = this.getCounter
+      type ? counter++ : counter--
+      this.setCounter(counter)
```

Comment:
counter is a function, then "var counter" again ??? please adjust the naming
this function is unnecessary long 
ex: 
changeCount(val){
this.setCounter(this.getCounter+val) ...
}

Reply:
this one not yet fixed ?